server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -115,8 +121,10 @@ private static boolean clusterUUIDCommitted(Object[] fields) {
         );
         PARSER.declareString(ConstructingObjectParser.constructorArg(), PREVIOUS_CLUSTER_UUID);
         PARSER.declareBoolean(ConstructingObjectParser.constructorArg(), CLUSTER_UUID_COMMITTED);
+        PARSER.declareString(ConstructingObjectParser.constructorArg(), GLOBAL_METADATA_FIELD);
```

Comment:
Since we are going to add codec version in manifest file as well, we are going to leverage codec version to handle this. 

If codec version is 2 [new version after introducing global metadata], then parser will parse those new field else it will use the older parser only.